### ISSUE TITLE
Support promscale extension 0.8.0

### ISSUE THREAD

**Issue by abcdabcd987** [timescale/promscale#1827]
**Describe the bug**
```
level=info ts=2023-01-09T18:39:14.083Z caller=extension.go:245 msg="using a higher extension version than expected" extension_name=promscale current_version=0.8.0 expected_version=0.7.0                                                                                                                                                             
level=error ts=2023-01-09T18:39:14.142Z caller=runner.go:127 msg="aborting startup due to error" err="migration error: failed to install/upgrade promscale extension: error encountered while migrating extension: problem checking promscale extension version: promscale extension is installed but on an unsupported version: 0.8.0. Expected: >=0.7.0 <0.7.99"
```

I believe this issue might have been fixed by [this commit](https://github.com/timescale/promscale/commit/97a6c139beaa619d2403b2a738718e3f5af1ff37), but can you release a Docker Image for it?

**To Reproduce**
Use the latest docker images:
- timescale/timescaledb-ha:pg14-latest: `timescale/timescaledb-ha@sha256:f576bcf8bb4ad146e0cc3c5ff28d43bed34e74ec1710571e53099d9399f88002`
- timescale/promscale:latest: `timescale/promscale@sha256:e9b697003cfe1cafcd2888cc0317d21069c3755fd6b69bab84f54054f94763c6`

**Expected behavior**
No error

**Version**
 - Distribution/OS: Docker Image
 - Promscale: 0.16
 - TimescaleDB: 2.9.1
 - Promscale Extension: 0.8.0

**Comment by ramonguiu** [comment]
@abcdabcd987 yes, we'll be releasing shortly the Promscale Connector version that is compatible with that extension.

Did you install that version of the extension manually? Typically we don't expect users to do that, the Promscale Connector takes care of upgrades to ensure you run a compatible version of the extension.

**Comment by abcdabcd987** [comment]
> Did you install that version of the extension manually?

Yes indeed. I pulled the latest `timescale/timescaledb-ha:pg14-latest`. After upgrading the `timescaledb` extension, I thought I might as well upgrade `promscale` extension.

As I reread the document about upgrading [promscale](https://docs.timescale.com/promscale/latest/guides/upgrade/), I realized that I shouldn't do that. Since this is not a bug, I'll close this comment. Thanks for the help!

**Comment by ramonguiu** [comment]
@abcdabcd987 heads up the promscale connector release is out: https://github.com/timescale/promscale/releases/tag/0.17.0
